packages/data-stores/src/automated-transfer-eligibility/types.ts

```diff
@@ -29,17 +31,11 @@ export interface TransferEligibilityWarningsType {
 export interface TransferEligibility {
 	[ index: number ]: TransferEligibility;
 
-	errors: TransferEligibilityError[];
+	errors: InternalTransferEligibilityError[];
 	is_eligible: boolean;
 	warnings: TransferEligibilityWarningsType;
 }
 
 export type State = {
 	[ key: number ]: TransferEligibility;
 };
-
-export interface StatusMapping {
```

Comment:
This type is useless since `statusMapping` itself provides a lot of detail as a const object.